gdal/frmts/wms/wmsutils.cpp

```diff
@@ -28,24 +28,26 @@
 
 #include "wmsdriver.h"
 
+CPL_CVSID("$Id$")
+
```

Comment:
https://github.com/OSGeo/gdal/pull/2503#issuecomment-626307246

Reply:
Yes, but I didn't want to show this file as modified, it was just busy fingers.

Reply:
Gotcha